Db/models service attachment feature

Chinmay891 opened the pull request:
Created three models for service page. Lock files not included.

Chinmay891 committed:
Add Service model and create migration for services table

Chinmay891 committed:
Add Attachment model and create migration for attachments table

Chinmay891 committed:
Add Feature model and create migration for features table

Chinmay891 committed:
Add migration files

Chinmay891 committed:
Add schema file

rutujanirmal commented on db/migrate/20220901115806_create_features.rb:
```diff
@@ -0,0 +1,9 @@
+class CreateFeatures < ActiveRecord::Migration[7.0]
+  def change
+    create_table :features do |t|
+      t.string :feature_name
```
it should be only name not feature_name

rutujanirmal commented on db/migrate/20220901113844_create_attachments.rb:
```diff
@@ -0,0 +1,9 @@
+class CreateAttachments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :attachments do |t|
+      t.string :image_url
```
rename this to attachement url to use it for other file types as well

Chinmay891 commented on db/migrate/20220901113403_create_services.rb:
```diff
@@ -0,0 +1,10 @@
+    create_table :services do |t|
+      t.string :service_name
+      t.string :service_info
+      t.string :video_url
```
We just have 3 video URLs for 3 services, so it will be more easy to populate them this way.

Chinmay891 commented on app/models/service.rb:
```diff
@@ -0,0 +1,4 @@
+class Service < ApplicationRecord
+    has_many :attachment
```
We give file name in has_many, plural form will refer to the table

rutujanirmal replied:
![image](https://user-images.githubusercontent.com/64598190/188554367-77486676-4dc1-4df3-907f-40f3ffb88e69.png)

Chinmay891 committed:
Add migrations for
 - update the column name of features table and attachments table
 - drop video url column from services table
 - add column to attachments table with enum type

Chinmay891 committed:
Add enum in attachment model
update schema file with new migrations

Chinmay891 committed:
Changed the has_many value to plural form

rutujanirmal commented on db/migrate/20220905102453_update_column_name.rb:
```diff
@@ -0,0 +1,6 @@
+class UpdateColumnName < ActiveRecord::Migration[7.0]
```
rollback and use proper naming for migrations

Chinmay891 committed:
Rename existing migrations
